rotkehlchen/tests/unit/test_tasks_manager.py

```diff
@@ -596,3 +596,24 @@ def test_augmented_detection_pendle_transactions(
 
     updated_token = cast(EvmToken, globaldb.resolve_asset(identifier=token.identifier))
     assert updated_token.protocol is None
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('max_tasks_num', [5])
+def test_maybe_update_underlying_assets(
+        task_manager: TaskManager,
+        globaldb: GlobalDBHandler,
+) -> None:
+    """Test that the task updating the underlying assets in globaldb works"""
+
+    # check the underlying assets table in globaldb before running the task
+    with globaldb.conn.read_ctx() as cursor:
+        initial_count = cursor.execute('SELECT COUNT(*) FROM underlying_tokens_list').fetchone()[0]
+
+    task_manager.potential_tasks = [task_manager._maybe_update_underlying_assets]
+    task_manager.schedule()
+    gevent.joinall(task_manager.running_greenlets[task_manager._maybe_update_underlying_assets])  # wait for the task to finish since it might context switch while running  # noqa: E501
+
+    # check the underlying assets table in globaldb after running the task
+    with globaldb.conn.read_ctx() as cursor:
+        assert cursor.execute('SELECT COUNT(*) FROM underlying_tokens_list').fetchone()[0] > initial_count  # noqa: E501
```

Comment:
It would be better to check that you create also the debt tokens and the a-tokens. Basically you increated the count by protocol=CPT_AAVE_V3. If we add the underlying tokens to the globaldb this test will break